src/commands/log/index.js

```diff
@@ -6,11 +6,11 @@ export const {parse, usage, commandDescriptor} = loadCommand('log')
 export const invoke = composeInvoke(parse, usage, (args, notify, options) => {
   const {
     lgJWT,
-    lgPlayer,
+    lgUser,
     formatError,
     formatMessage,
   } = options
-  if (!lgJWT || !lgPlayer || !lgPlayer.id) {
+  if (!lgJWT || !lgUser || lgUser.roles.indexOf('player') < 0) {
```

Comment:
Nitpick: a tiny bit antsy antsy about the lack of defensive checking on the type of `lgUser.roles)` given that there's nothing I know of that takes any measures to ensure that the value of the `roles` property on a user object is an array. I have some idea for changing this in our javascript apps (we might consider using [joi](https://github.com/hapijs/joi), for example), but in the meantime...